quickstarts/gcp/gcp-memcached/config.yml

```diff
@@ -0,0 +1,36 @@
+id:
+slug: gcp-memcached
+description: |-
+  ## What is Google Memcached?
+
+  Memorystore for Memcached is a fully managed Memcached service for Google Cloud. Applications running on Google Cloud can achieve extreme performance by leveraging the highly scalable, available, secure Memcached service without the burden of managing complex Memcached deployments.
+
+  ### Get started!
+
+  Start monitoring Memcached by connecting Google Cloud Platform (GCP) to New Relic!
+  Check out our Memcached documentation to instrument your cloud service and manage the stability, scalability, and reliability of your systems with 
+  New Relic's infrastructure monitoring capabilities.
+
+  ### More info
+
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/google-cloud-platform-integrations/gcp-integrations-list/google-memorystore-memcached/) to learn more about New Relic monitoring for Memcached.
+summary: Monitor Memcached by connecting GCP to New Relic
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: Memcached
```

Comment:
https://github.com/newrelic/newrelic-quickstarts/blob/a01923069e2662c87c5666c6dc175c48dc2a9f3b/quickstarts/memcached/config.yml#L3 

this quickstart is already using this name